### ISSUE TITLE
Home Assistant restart shows ring entities as unavailable

### ISSUE THREAD

**Issue by mcblok** [tsightler/ring-mqtt#112]
I use the Docker image and upgraded from 3.3.0 to 4.0 and noticed that every time I restart Home Assistant all Ring entities become unavailable. Restarting the ring-mqtt Docker container is needed. Version 3.3.0 does not have this issue, it temporarily shows the entities as unavailable but in a few seconds they show up again.

**Comment by tsightler** [comment]
Did you follow the instructions in the Readme for Home Assistant >0.113 and update the birth topic from hass/status to homeassistant/status?  4.0 drops support for the previous in favor of the new HA default behavior.

**Comment by mcblok** [comment]
Thanks, that was the issue. Not sure why it worked in 3.3.0. And thanks for making ring-mqtt available, it's great!

**Comment by tsightler** [comment]
Yeah, in versions prior to 4.0 the HA state topic was configurable, and the default config was to use the hass/status since that was what was used as the example in the Home Assistant docs back when I first developed the script so, if you just followed the HA docs and nothing else it would "just work".  However, in Home Assistant 0.113 and above they made birth messages enabled by default, without any special configuration, but it used the homeassistant/status.  To make new users "just work" I needed to change the default to the same as the Home Assistant new defaults, and 4.0 implemented that change.

The unfortunate side effect of that change is that it breaks anyone who was using the old Home Assistant configuration from the docs with the old script defaults.  I tried to document this as best possible, but perhaps it could be worded better.

I considered just monitoring both the old and new topics for a while, and perhaps I should have done that, but my experience says that just pushes the problem into some point in the future and I thought, it's 4.0, with a lot of changes, so why not bite the bullet now.

Basically, Home Assistant made the decision to minimize the use of the "hass" abbreviation in favor of the full Home Assistant name pretty much everywhere.  The good news is, I think this is unlikely to ever change again since it's now the default behavior for every single HA install as it would risk breaking too many new users that just leave everything default.

Thanks for the support!
